interfaces/builtin/serial_port.go

```diff
@@ -109,23 +164,63 @@ func (iface *SerialPortInterface) PermanentPlugSnippet(plug *interfaces.Plug, se
 
 // ConnectedPlugSnippet returns security snippet specific to the plug
 func (iface *SerialPortInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
+
+	slotPath, slotOk := slot.Attrs["path"].(string)
+	if !slotOk || slotPath == "" {
+		// don't have path attribute so must be using Udev tagging
+
+		switch securitySystem {
+		case interfaces.SecurityAppArmor:
+			// Wildcarded apparmor snippet as the cgroup will restrict down to the
+			// specific device
+			return []byte("/dev/tty* rw,\n"), nil
```

Comment:
Thank you for using this instead of `/dev/** rw,`. There is a bug in snap-confine that I need to fix wrt no matched tags.